### PR TITLE
outputs current local weather

### LINKED ISSUE #10: [LCD-007] Output local weather onto LCD
Example scenario:
Brother "What is the weather?"
Robot's LCD "It is sunny at 78F "

Dependencies:
* Mech needs to have finished wiring the LCD
* Mech should also verify LCD is working after wiring is completed
* Established and stable data source for weather

Requirements:
* Acquire robot's geolocation
* Update the voicecommand config file to fetch a response from that data source
* Output response onto LCD/command line

Suggestion (you can follow or ignore): Probably make a python script that fetches the next event & relevant data from a data source. The corresponding voicecommand cmd should be updated to invoke this script, then output onto LCD/command line -Joey

Relevant: https://github.com/ThetaTauSBU/OT_mascot_robot/wiki/Detailed-Project-Plan#milestone-3-lcd-response
https://maker.pro/raspberry-pi/tutorial/the-best-voice-recognition-software-for-raspberry-pi

### PR DESCRIPTION
**Pull request by jahnaviarora00**
# Summary
Script that outputs the current local weather (description, temp, humidity, pressure etc.) for a current city. 
Need to install requests (sudo pip install requests)

# Test Plan
Reviewer can change city to test code. 

# Test Data
tested with multiple cities
ex. 
input - Stony Brook 
output - 
City: Stony Brook
Temperature (F): 39.26
Humidity: 45
Pressure: 1034
Weather Report: few clouds

input - Syosset
output - 
City: Syosset
Temperature (F): 39.6
Humidity: 52
Pressure: 1034
Weather Report: clear sky

### CODE REVIEW

**Review by SeannZhou — APPROVED** [review]

File: scripts/LCD-007.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import requests, json
```
Comment (gammaseeker) [inline scripts/LCD-007.py:1]: Add new line after import statement

File: scripts/LCD-007.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+BASE = "https://api.openweathermap.org/data/2.5/weather?"
+APIKEY = "fc2822e01a0efcce52cb56212fe44fd5"
+CITY = "Stony Brook"
+URL = BASE + "q="+ CITY + "&appid=" + APIKEY
```
Comment (gammaseeker) [inline scripts/LCD-007.py:5]: Add new line after constant declarations

File: scripts/LCD-007.py [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+    humidity = main['humidity']
+    pressure = main['pressure']
+    report = data['weather']
+    print("City: {}".format(CITY))
+    print("Temperature (F): {}".format(tempF))
+    print("Humidity: {}".format(humidity))
+    print("Pressure: {}".format(pressure))
+    print("Weather Report: {}".format(report[0]['description']))
+else:
+    print("Error in the HTTP Request")
```
Comment (gammaseeker) [inline scripts/LCD-007.py:21]: Add a new line at the end of the file, it seems like github is complaining about it.

**Review by gammaseeker — CHANGES_REQUESTED** [review]
Instead of `LCD-007.py` let's rename your file to `get_weather.py` to make it more descriptive. Everything else looks good other than some nit picky styling changes.

File: scripts/LCD-007.py (hunk 1 above)
Comment (gammaseeker) [inline scripts/LCD-007.py:1]: Add shebang line before import statement `#!/usr/bin/python3`
This way voicecommand can simply invoke your script without having to call `python3`

File: scripts/LCD-007.py [hunk 4]
```diff
@@ -0,0 +1,21 @@
+import requests, json
+BASE = "https://api.openweathermap.org/data/2.5/weather?"
+APIKEY = "fc2822e01a0efcce52cb56212fe44fd5"
```
Comment (gammaseeker) [inline scripts/LCD-007.py:3]: General comment, you don't need to take action on this. It is bad practice to put API keys in the code. Let's say you are working with sensitive data or have a certain quota to the amt of times you can send a request to an API. If I can see your API key in the code I can grief your program and abuse your credentials to hit that quota. As a result of hitting that quota, I render your service useless bc you can't use that API anymore. 

The canonical way to handle this problem is to have a hidden `.env` file somewhere that stores all your API keys and other credentials. You'd have your script read from this `.env` file and store your credentials in some constant so nobody can know your API key. Obviously, this `.env` file is never pushed to github or uploaded anywhere.

**Pull request closed without merging**